src/main/webapp/WEB-INF/faces-config.xml

```diff
@@ -13,6 +13,10 @@
             <supported-locale>fr</supported-locale>
             <supported-locale>zh_CN</supported-locale>
         </locale-config>
+        <resource-bundle>
+            <base-name>/Bundle_trsa_registry</base-name>
+            <var>bundle_trsa_registry</var>
+        </resource-bundle>
```

Comment:
Why is this entry necessary and will it work with all the internationalization code Scholars Portal has been working on?